runtime/src/accounts_db.rs

```diff
@@ -489,6 +491,8 @@ pub struct AccountsDbConfig {
     pub ancient_append_vec_offset: Option<i64>,
     pub skip_initial_hash_calc: bool,
     pub exhaustively_verify_refcounts: bool,
+    /// when stakes cache consistency check occurs, assert that cached accounts match accounts db
+    pub assert_stakes_cache_consistency: bool,
```

Comment:
how does one overrides this to true?

Reply:
`AccountsDbConfig` has a member.

Reply:
right, and i am asking when i start the node how can i override this? :)

Reply:
there is no cli arg to enable this. It was designed to always be enabled for all tests. Feel free to add one.